visits-service-new/src/main/java/com/petclinic/visits/visitsservicenew/VisitsServiceNewApplication.java

```diff
@@ -5,7 +5,6 @@
 import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
-@ComponentScan("com.petclinic")
```

Comment:
Not necessary since it is included in the `@SpringBootApplication` annotation. This duplication of annotation creates ApplicationContext issues with unitTests unless removed.